config/clusters/callysto/common.values.yaml

```diff
@@ -37,9 +32,6 @@ jupyterhub:
           name: Callysto
           url: https://www.callysto.ca
   singleuser:
-    image:
```

Comment:
I consistently see PRs bumping the image tag from @ianabc (https://github.com/2i2c-org/infrastructure/pull/2823). Did you switch to using the configurator, @ianabc?

Reply:
I used the configurator for the last two of image updates we did. I remember where we discussed this, but I thought this was the intention, but I haven't been following the prePuller discussions so if I'm messing something up, please let me know and I'll change it.